pkg/flowkit/state.go

```diff
@@ -51,10 +51,29 @@ type Contract struct {
 
 // State manages the state for a Flow project.
 type State struct {
-	conf         *config.Config
-	confLoader   *config.Loader
-	readerWriter ReaderWriter
-	accounts     *Accounts
+	conf             *config.Config
+	confLoader       *config.Loader
+	readerWriter     ReaderWriter
+	accounts         *Accounts
+	accountLocations map[string]string
```

Comment:
I would make this be a map with accounts, so:
```
accountLocations map[*Account]string
```
And then all the changes that come with this